src/components/layout/__tests__/ButtonPlus.test.tsx

```diff
@@ -4,31 +4,25 @@ import "jest-styled-components";
 
 import ButtonPlus from "components/layout/ButtonPlus";
 
-describe("button default image", () => {
-  test("Button render", () => {
+describe("ButtonPlus component", () => {
+  test("Button render plus", () => {
     render(<ButtonPlus />);
     const testImage = document.querySelector("img") as HTMLImageElement;
     expect(testImage.alt).toContain("plus icon");
   });
-});
 
-describe("button minus image", () => {
-  test("Button render", () => {
+  test("Button render minus", () => {
     render(<ButtonPlus plusIcon={false} />);
     const testImage = document.querySelector("img") as HTMLImageElement;
     expect(testImage.alt).toContain("minus icon");
   });
-});
 
-describe("button default bg-color", () => {
-  test("Button render", () => {
+  test("Button default bg-color", () => {
     const tree = renderer.create(<ButtonPlus />).toJSON();
     expect(tree).toHaveStyleRule("background-color", "#95E1D3");
```

Comment:
instead of pasting the color as string I recommend creating constants for the colors at the top of the test